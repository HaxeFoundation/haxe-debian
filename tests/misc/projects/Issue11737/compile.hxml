--main Main
--java bin
-D jvm
