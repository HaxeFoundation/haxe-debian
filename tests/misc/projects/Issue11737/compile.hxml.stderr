Warning : (WDeprecated) --java out.jar -D jvm is deprecated; use --jvm out.jar directly
